service-stubs/identity/org.wso2.carbon.identity.sts.passive.stub/src/main/resources/IdentityPassiveSTSService.wsdl

```diff
@@ -138,6 +130,7 @@
                     <xs:element minOccurs="0" name="context" nillable="true" type="xs:string"/>
                     <xs:element minOccurs="0" name="currentTimeAtRecepient" nillable="true"
                                 type="xs:string"/>
+                    <xs:element minOccurs="0" name="dialect" nillable="true" type="xs:string"/>
```

Comment:
WSDL has not been updated correctly in a previous commit. See "Re: [Commits] [Carbon-platform] svn commit r146553 - in carbon/platform/branches/4.0.0/components/identity/org.wso2.carbon.identity.sts.passive/4.0.1/src/main/java/org/wso2/carbon/identity/sts/passive: . processors" in commits mails